From 75a5e149551dd0c4a89d6fd1902d39dd9fae05bb Mon Sep 17 00:00:00 2001
From: Jeff King <peff@peff.net>
Date: Wed, 11 Mar 2020 18:11:37 -0400
Subject: t/lib-credential: use test_i18ncmp to check stderr

The credential tests have a "check" function which feeds some input to
git-credential and checks the stdout and stderr. We look for exact
matches in the output. For stdout, this makes sense; the output is
the credential protocol. But for stderr, we may be showing various
diagnostic messages, or the prompts fed to the askpass program, which
could be translated. Let's mark them as such.

(cherry picked from commit 17f1c0b8c7e447aa62f85dc355bb48133d2812f2)
Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 t/lib-credential.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/lib-credential.sh b/t/lib-credential.sh
index d8e41f7ddd..fad09c9d1f 100755
--- a/t/lib-credential.sh
+++ b/t/lib-credential.sh
@@ -19,7 +19,7 @@ check() {
 		false
 	fi &&
 	test_cmp expect-stdout stdout &&
-	test_cmp expect-stderr stderr
+	test_i18ncmp expect-stderr stderr
 }
 
 read_chunk() {
-- 
2.26.0.292.g33ef6b2f38

